opentelemetry/src/main/scala/zio/telemetry/opentelemetry/package.scala

```diff
@@ -0,0 +1,9 @@
+package zio.telemetry
+
+import io.opentelemetry.api.trace.StatusCode
+
+package object opentelemetry {
+
+  type ErrorMapper[E] = PartialFunction[E, StatusCode]
```

Comment:
I would inline this type alias, as it makes the code harder to understand, I think (by adding 1 layer of indirection).

Reply:
Yeah, I came up with this decision in my latest changes - I added a `case class ErrorMapper[E](body: PartialFunction[E, StatusCode])`. I guess that's what you mean.